src/Conda.jl

```diff
@@ -191,27 +189,31 @@ _quiet() = get(ENV, "CI", "false") == "true" ? `-q` : ``
 "Install miniconda if it hasn't been installed yet; _install_conda(true) installs Conda even if it has already been installed."
 function _install_conda(env::Environment, force::Bool=false)
     if force || !isfile(Conda.conda)
+        # This assumes that the conda executable will in some directory under
+        # CONDA_EXE_PREFIX.
+        # Ex: If CONDA_EXE="$HOME/miniforge3/bin/conda", then CONDA_EXE_PREFIX="$HOME/miniforge3"
+        CONDA_EXE_PREFIX = conda |> dirname |> dirname
```

Comment:
I think this assumption is too strong. It's possible that people use a shim script or symlink. Can you use something like `conda info --json` or `conda run python -c 'import sys; print(sys.executable)'`?

Nitpicks: It's strange that a local variable is all caps. I also notice that src/Conda.jl does not use `|>` for unary function application. I have nothing against `|>` per se, but, I think it's good to follow the style of surrounding code.

Reply:
@mkitti, why do we need a `CONDA_EXE_PREFIX` when `PREFIX` should work just as fine?

Reply:
`PREFIX` is determined by `ROOTENV`. https://github.com/JuliaPy/Conda.jl/blob/f1690184f15c875100e35dde187f3dd2a1235db8/src/Conda.jl#L47

We reach this line of code when the conda executable does not exist at `Conda.conda`.
Thus, the objective here is to install the conda executable at the location specified by `CONDA_JL_CONDA_EXE` / `CONDA_EXE` / `Conda.conda`.

Reply:
You mean when the user sets `CONDA_JL_CONDA_EXE` env variable to a non-existent conda executable? In that case, we should just error in `deps.jl`.

Reply:
We can add a check at build time. What should we do at runtime when `_install_conda` is invoked?

Reply:
> We can add a check at build time.

Thanks

> What should we do at runtime when _install_conda is invoked?

Nothing. We just use PREFIX as the if condition above is invoked only if `CONDA_JL_CONDA_EXE` is not set.

Reply:
Is it only invoked if `CONDA_JL_CONDA_EXE` is not set? It might also be invoked if `CONDA_EXE` gets deleted between build time and run time.

Reply:
I forgot to mention another point: does Conda.jl still use `PREFIX`? If not, it's better to remove it or at least deprecate it.

Reply:
There's an internal constant called `PREFIX`, but it does not use an environmental variable called PREFIX.

Reply:
> Is it only invoked if CONDA_JL_CONDA_EXE is not set? It might also be invoked if CONDA_EXE gets deleted between build time and run time.

True. I'm not sure how to handle it. If `CONDA_EXE` gets deleted between build time and run time, I'm not sure if we should install it there.